src/components/content/project_components/View/View.js

```diff
@@ -4,7 +4,56 @@ import ProjectSetupSidebar from '../ProjectSetupSidebar/ProjectSetupSidebar'
 import './view.scss';
 
 class View extends Component {
+  constructor(props){
+  super(props);
+  this.state={
+      view: {
+          arr:[{
```

Comment:
`arr` is redundant, the default value for `view` should be an empty array, which will be populated when the component renders. I've attached a link to a completed component to assist you.

https://github.com/Appr/Appr/blob/master/src/components/content/project_components/Ideas_Users/Ideas_Users.js

Reply:
Hey it looks like Garry finished up this component, we're going to merge that, so when you do another pull from the master to resolve merge conflicts, be sure to accept the `incoming changes` for that.

Reply:
These changes need to be reversed